muser/src/main/java/edu/usf/cutr/mfe/ProcessorMain.java

```diff
@@ -0,0 +1,49 @@
+package edu.usf.cutr.mfe;
```

Comment:
We're using `edu.usf.sas.pal.muser` for the MUSER Android app, so let's keep that consistent here